### ISSUE TITLE
STX balance does not take into consideration locked balance

### ISSUE THREAD

**Issue by aulneau** [leather-io/extension#1273]
When sending STX, if the account is stacking, the "send max" and balance displays are of the total amount, but does not consider the difference.

**Comment by Eshwari007** [comment]
@aulneau should the use case be tested:

-When sending STX, if the account is stacking in desktop wallet the "send max" and balance displays are of the total amount, but does not consider the difference in webwallet.
Should I cross check both of the places ?

**Comment by aulneau** [comment]
> but does not consider the difference in webwallet.
can you rephrase?

the values should be the same between the web wallet and the desktop wallet. meaning:
- user is stacking 500 STX
- user balance is 1500 STX 
- user goes to send page
- user hits send max
- value should be slightly less than 1000 STX (minus network fees, so 999.999 for example)

**Comment by Eshwari007** [comment]
Test passed hence moving it to QA Done
